### ISSUE TITLE
Recursive function

### ISSUE THREAD

**Issue by Kr00l** [WaynePhillipsEA/twinbasic#77]
**Describe the bug**
The below code does not work, while it works in VB6/VBA.
The function makes a one-time recursion of itself.
```
Private Function MyTest(Optional ByRef B As Boolean = False) As Boolean
If B = False Then MyTest = MyTest(True) Else MyTest = True
End Function
```

**To Reproduce**
`MsgBox MyTest()`

**Expected behavior**
Function should compile and return True.

**Screenshots**
![Untitled](https://user-images.githubusercontent.com/33756666/115386738-1445e700-a1da-11eb-916c-48854b3f00a0.png)

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10]
 - twinBASIC compiler version [e.g. 0.9.1750]

**Additional context**

**Comment by WaynePhillipsEA** [comment]
Thanks for reporting.  Looks like a resolver bug, as the MyTest(true) call is binding to the MyTest return value local variable.   For now, prefix with the module name qualifier, e.g.    ModuleName.MyTest(True)   This will be fixed in a couple of days.

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.9.2100.  Thanks for reporting!
